### PR TITLE
add quotes rule

### PR DESCRIPTION
**Pull request by kellyselden**
Closes #42.

This a MVP of the quotes rule that can be built upon if necessary.

### CODE REVIEW

File: docs/rule/quotes.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#### eol-last
```
Comment (rwjblue) [inline docs/rule/quotes.md]: Seems like a copy/paste error...

File: lib/rules/lint-quotes.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+    throw new Error(errorMessage);
+  }
+
+  visitor() {
+    let badChar;
+    let message;
+    switch (this.config) {
+    case 'double':
+      badChar = '\'';
```
Comment (rwjblue) [inline lib/rules/lint-quotes.js]: Thoughts on using a template string here instead to avoid the escaping?

Comment (kellyselden) [reply lib/rules/lint-quotes.js]: I think I got a linting error when I tried that.

Comment (kellyselden) [reply lib/rules/lint-quotes.js]: yep "Strings must use singlequote."

File: docs/rule/quotes.md (hunk 1 above)
Comment (kellyselden) [reply docs/rule/quotes.md]: fixed.

**Comment by btecu** [comment]
https://github.com/ember-template-lint/ember-template-lint/pull/42#issuecomment-218965107 would be nicer. It's nice to be able to have:
```
<div class="some-component" {{action 'doSomething'}}>
  ...
</div>
```

I believe right now it enforces one type of quotes and defaults to double quotes.

**Comment by rwjblue** [comment]
Yep, confirm

**Pull request merged**